processor/src/test/java/org/robolectric/annotation/processing/validator/InDevelopmentValidatorTest.java

```diff
@@ -65,7 +65,7 @@ public void implementationWithoutInDevelopmentFailsToCompiles() {
     assertAbout(singleClass(props, getClassRootDir(Dummy.class), unreleased.getSdkInt()))
         .that(testClass)
         .failsToCompile()
-        .withErrorContaining("No such method in com.example.objects.Dummy for SDK 35")
+        .withErrorContaining("No such method doSomething() in com.example.objects.Dummy for SDK 35")
```

Comment:
In case my previous comment makes sense:
```suggestion
        .withErrorContaining("No method doSomething() in com.example.objects.Dummy for SDK 35")
```